client/asset/dcr/dcr.go

```diff
@@ -2502,6 +2502,15 @@ func (dcr *ExchangeWallet) sendCoins(addr stdaddr.Address, coins asset.Coins, va
 	return tx, uint64(txOut.Value), err
 }
 
+// newTxOut returns a new transaction output with the given parameters.
+func newTxOut(amount int64, pkScriptVer uint16, pkScript []byte) *wire.TxOut {
+	return &wire.TxOut{
+		Value:    amount,
+		Version:  pkScriptVer,
+		PkScript: pkScript,
+	}
+}
```

Comment:
@davecgh Suggestion taken regarding `wire.TxOut` construction, but I forgot to ask at the time of your suggestion why the wire.NewTxOut constructor doesn't get updated instead?  Is that planned?